samples/repo-search/desktopApp/src/jvmMain/kotlin/app/cash/paging/samples/reposearch/Main.kt

```diff
@@ -0,0 +1,176 @@
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import androidx.paging.LoadState
+import androidx.paging.PagingData
+import app.cash.paging.LazyPagingItems
+import app.cash.paging.collectAsLazyPagingItems
+import app.cash.paging.items
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.emitAll
+import kotlinx.coroutines.flow.emptyFlow
+import kotlinx.coroutines.launch
+import kotlin.coroutines.CoroutineContext
+
+fun main() {
+  application {
+    val windowState = rememberWindowState()
+
+    Window(
+      onCloseRequest = ::exitApplication,
+      state = windowState,
+      title = "PagingComposeDesktop"
+    ) {
+
+      val events = MutableSharedFlow<Event>(extraBufferCapacity = Int.MAX_VALUE)
+      val viewModels = MutableStateFlow<ViewModel>(ViewModel.Empty)
+
+      val presenter = RepoSearchPresenter()
+
+      val scope = rememberCoroutineScope()
+
+      // TODO("OG") Fix
+      // Cannot access class 'kotlin.coroutines.CoroutineContext'.
+      // Check your module classpath for missing or conflicting dependencies
+      scope.launch {
+        viewModels.emitAll(presenter.produceViewModels(events))
+      }
+
+      // TODO("OG") Fix
+      // Cannot access class 'kotlin.coroutines.CoroutineContext'.
+      // Check your module classpath for missing or conflicting dependencies
+      val viewModel by viewModels.collectAsState()
+      RepoSearchTheme {
+        RepoSearch(viewModel) { event ->
+          events.tryEmit(event)
+        }
+      }
+    }
+  }
+}
+
+@Composable
```

Comment:
All the compose/ui here is a copy-paste from the android sample app. As a good POC for KMP, the UI should also move to a common module